aks/postgres/variables.tf

```
@@ -111,3 +111,8 @@ variable "azure_maintenance_window" {
   })
   default = null
 }
+
+variable "enable_storage_account_for_backups" {
```

```suggestion
variable "azure_backup_storage_account" {
```

Anything that's specific to Azure I've prefixed with `azure` to help document the behaviour behind the variable. The current name suggests it could create a backup storage account for databases backed by Kubernetes.

---

changed as suggested